spec/integrity.yml

```diff
@@ -292,7 +295,7 @@ data/type3_font.pdf:
   :md5: ef2a3eb57d5996c4a33681e1aec98f2c
 data/type3_font_with_rare_font_matrix.pdf:
   :bytes: 11541
-  :md5: 08f69084d72dabc5dfdcf5c1ff2a719f
+  :md5: '08f69084d72dabc5dfdcf5c1ff2a719f'
```

Comment:
Out of interest, what version of ruby (and what YAML lib) are you using? It'd be helpful if PR diffs didn't include unrelated changes like these.

Reply:
Sorry I'm on Ruby 2.3.3p222. And I think YAML is included in Ruby, I can't see it as a gem.

But yes will revert those unrelated changes

Reply:
I think YAML is just adding quotes around strings that start with "0", in case they are parsed as octals.